src/rust/engine/protos/protos/pants/sandboxer.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package pants.sandboxer;
+
+import "build/bazel/remote/execution/v2/remote_execution.proto";
+
+service SandboxerGrpc {
+  rpc MaterializeDirectory (MaterializeDirectoryRequest) returns (MaterializeDirectoryResponse);
+}
+
+message MaterializeDirectoryRequest {
```

Comment:
Maybe add doc comments for all of these messages and fields?

Reply:
Done